packages/api/amplify_api/lib/src/graphql/web_socket/blocs/web_socket_bloc.dart

```diff
@@ -462,7 +463,8 @@ class WebSocketBloc with AWSDebuggable, AmplifyLoggerMixin {
     _currentState.service.close();
 
     await Future.wait<void>([
-      _networkSubscription.cancel(),
+      // TODO(elijah): <link to connectivity plus ticket>
+      if (zIsWeb || !Platform.isWindows) _networkSubscription.cancel(),
```

Comment:
question: Did we try adding `.ignore` to this call `.cancel().ignore()`? I'm not suggesting that as a change, just curious.

Reply:
I'm okay with `.cancel().ignore()` as well.